src/core/loaders/Loader.ts

```diff
@@ -11,6 +11,8 @@ export abstract class Loader extends Disposable {
   protected _onDidChange = new EventEmitter<string>()
   readonly onDidChange = this._onDidChange.event
 
+  readonly pluralizationKeysV4 = ['one', 'other', 'zero', 'two', 'few', 'many']
```

Comment:
Why v4? Does this come from the comment on `derivedKeyRules` where it calls these key names the "v4 format"?

Reply:
Good point ... I took the `v4` from the `i18next` [convention](https://www.i18next.com/misc/json-format#i18next-json-v4) but then needed to put the change into Loader instead of a specific framework ... might need a better name (or just drop the v4 part)